blobstorage/values.yaml

```
@@ -0,0 +1,6 @@
+location: uksouth
+resourceGroup: someGroup
+setup:
+  containers:
+   - first-container
```

probably should just default to one container, 
maybe 
```suggestion
   - container
```